samples/python_interop/qiskit.ipynb

```diff
@@ -0,0 +1,480 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "ae56fce0",
+      "metadata": {},
+      "source": [
+        "# Q# Interop with Qiskit"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "2b838c23",
+      "metadata": {},
+      "source": [
+        "The modern QDK provides interoperability with Qiskit circuits built upon the core Q# compiler infrastructure.\n",
+        "\n",
+        "This core enable integration and local resource estimation without relying on external tools. Users are able to estimate resources for their Qiskit circuits locally (see the [resource estimation with Qiskit sample notebook](../../estimation/estimation-qiskit.ipynb)), leveraging the Q# compiler's capabilities for analysis, transformation, code generation, and simulation. This also enables the generation of QIR from Qiskit circuits leveraging the [modern QDKs advanced code generation capabilities](https://devblogs.microsoft.com/qsharp/integrated-hybrid-support-in-the-azure-quantum-development-kit/).\n",
+        "\n",
+        "This includes support for circuits with classical instructions available in Qiskit such as for loops, if statements, switch statements, while loops, binary expresssions, and more."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "4f761649",
+      "metadata": {},
+      "source": [
+        "## Running Qiskit circuits\n",
+        "The `QSharpBackend` backend is the main class to interact with for running circuits and generating QIR.\n",
+        "\n",
+        "To start, we'll set up a simple circuit with a prepared state."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "e91dd2d7",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qiskit import QuantumCircuit\n",
+        "import numpy as np\n",
+        "\n",
+        "circuit = QuantumCircuit(2, 2)\n",
+        "circuit.name = \"state_prep\"\n",
+        "\n",
+        "# State vector to initialize: |ψ⟩ = (|0⟩ - |1⟩) / √2\n",
+        "circuit.initialize([1 / np.sqrt(2), -1 / np.sqrt(2)], 0)\n",
+        "circuit.h(0)\n",
+        "circuit.measure(0, 0)\n",
+        "\n",
+        "circuit.prepare_state([1 / np.sqrt(2), -1 / np.sqrt(2)], 1)\n",
+        "circuit.h(1)\n",
+        "circuit.measure(1, 1)\n",
+        "\n",
+        "circuit.draw(output=\"text\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "922f8420",
+      "metadata": {},
+      "source": [
+        "With the circuit created, we can run the circuit with Q#'s backend. By default, it will use the `Unrestricted` profile meaning anything it allowed for simulation."
```

Comment:
```suggestion
        "With the circuit created, we can run the circuit with Q#'s backend. By default, it will use the `Unrestricted` profile meaning anything is allowed for simulation."
```